### PR TITLE
feat: Added training data generation script using YOLO + PaddleOCR + DeepSeek R1

### PR DESCRIPTION
**Pull request by Ghaadyy**
## Description

- Added a script used to generate training data to fine-tune _SeeClick_ on additional data

## Pipeline
YOLO object detection -> PaddleOCR to extract the text -> DeepSeek R1 to generate descriptions

Result: A list of images with descriptions and bounding boxes which can be fed into _SeeClick_ for better accuracy.

### CODE REVIEW

**Review by A-Karam20 — APPROVED** [review]
LGTM.
